core/src/components/modal/usage/angular.md

```diff
@@ -73,9 +73,10 @@ import { ModalPage } from '../modal/modal.page';
   styleUrls: ['./modal-example.css']
 })
 export class ModalExample {
-  constructor(public modalController: ModalController) {
+  // The `ion-modal` element reference.
+  modal: HTMLElement;
```

Comment:
Are we able to write docs on how to get a reference to the modal element? I worry that this functionality is a bit hidden in the usage examples by itself